specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -337,6 +337,88 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/offlineRegion": {
+      "post": {
+        "operationId": "DatabaseAccounts_OfflineRegion",
+        "x-ms-examples": {
+          "CosmosDBDatabaseAccountOfflineRegion": { "$ref": "./examples/CosmosDBDatabaseAccountOfflineRegion.json" }
+        },
+        "description": "Offline the specified region for the specified Azure Cosmos DB database account.",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "regionParameterForOffline",
+            "in": "body",
+            "required": true,
+            "description": "Cosmos DB region to offline for the database account.",
+            "schema": {
+              "$ref": "#/definitions/RegionForOnlineOffline"
+            }
+          }
+        ],
+        "responses": {
```

Comment:
Add a "default" response to capture all the error responses. The schema complies with ARM RPC error contract. Example can be found Batch RP swagger spec.